legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-core/src/main/java/org/finos/legend/engine/persistence/components/relational/api/RelationalIngestorAbstract.java

```diff
@@ -435,7 +506,7 @@ private List<IngestorResult> performFullIngestion(RelationalConnection connectio
         }
 
         // Evolve Schema
-        evolveSchema();
+        evolveMainDatasetSchema();
```

Comment:
we should call evolve() here to trigger both

Reply:
One problem here tho - if someone calls performFullIngestion, it seems like we will not record down the schema evol SQLs for metadata table in the IngestorResult

Reply:
Yes, will invoke evolve and return the results in the IngestorResult